src/test/store/receive-profile.test.js

```diff
@@ -221,45 +241,47 @@ describe('actions/receive-profile', function() {
   });
 
   describe('retrieveProfileFromAddon', function() {
-    let geckoProfiler;
-    let clock;
-
-    beforeEach(function() {
```

Comment:
I converted this from `beforeEach` to our current `setup` style. And I also converted from sinon fake timers to jest fake timers because it was easy.